superx/information_extractors/branch_info_extractor.py

```diff
@@ -152,10 +170,26 @@ def fill_branch_table(self, xml_info_list):
         branch_list = []
 
         for branch_id, branch_name, address, sub_chain_id, city in xml_info_list:
-            branch_list.append(Branch(id=branch_id,
-                                      name=branch_name, address=address,
-                                      sub_chain_id=sub_chain_id,
-                                      city=city,
-                                      chain_id=self.current_super['chain_id']))
+            # if not in db then add it
+            if (int(branch_id), self.current_super['chain_id']) not in self.branch_unique_constraint_set: # pylint: disable=line-too-long
```

Comment:
You can't disable every check that you don't like. Why not break this line into two so it is less the 80 chars?

Reply:
this hinders the readablity of this line. the reason I keep disabling the pylint error is only because of the readability and also since this still conforms with the PEB8 style it is ok.  
But I understand the comment and will make this change or now